oneflux_steps/ustar_cp_refactor_wip/cpdEvaluateUStarTh4Season20100901.m

```diff
@@ -61,136 +61,92 @@
 %	========================================================================
 %	========================================================================
 
-%	Initializations
+	nSeasons = 4; nStrataN = 4; 

```

Comment:
These numbers get defined repeatedly around the code - in cpdBootstrap, cpdAssign and here. For the Python implementation should be define a dataclass to store them which can then be accessed where needed?

Reply:
Yes, I think that would be good